src/rospkg/rospack.py

```diff
@@ -31,8 +31,13 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 import os
+import sys
 from threading import Lock
-from xml.etree.cElementTree import ElementTree
+
+if sys.version_info >= (3, 3):
+    from xml.etree.ElementTree import ElementTree
+else:
+    from xml.etree.cElementTree import ElementTree
```

Comment:
Please change the logic to avoid the version condition. Simply try to import `cElementTree` first and if that raises an `ImportError` fallback to the alternative.

Reply:
Thanks, made the changes.